test/CentOS-Linux-AppStream.repo

```diff
@@ -7,5 +7,6 @@
 name=CentOS Linux $releasever - AppStream
 mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=AppStream&infra=$infra
 #baseurl=http://mirror.centos.org/$contentdir/$releasever/AppStream/$basearch/os/
-gpgcheck=0
-enabled=1
\ No newline at end of file
+gpgcheck=1
```

Comment:
why do we need a gpg check btw? why do we need gpg at all?

Reply:
@eranha

Reply:
@sigalsax To verify the downloaded RPMs